text_collector_examples/deb_check_package_updates.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+/usr/bin/apt-get update
```

Comment:
I agree, let's not call apt-get update by default. I'd also prefer to let users do this on their own, but I'd be fine to have a flag to enable this for convenience (should be disabled by default).